docker/tpm_server.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=TPM Server
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/tpm_server
```

Comment:
You might need a remanufacture flag here `-rm`

This is what we do when we run in CI:

https://github.com/keylime/keylime/blob/89e2ffb55829ea3af7cb602b7df90e43d23372e1/scripts/tpm_serverd#L17

It might be ok as its going to be the first time it starts, but just in case you hit issues you can try the above.

Reply:
Good point. Almost all usage will be starting a new container, but in case you ever need to restart a container, or debug things within the container restarting services it's good to have. Added.